### DISCUSSION TITLE
Incoming confirmed not showing up in balance

### DISCUSSION THREAD

**Discussion by atg906** [Chia-Network/chia-blockchain#6197] [Support]
Received a confirmed amaount of 1.75 yesterday evening, yet still not showing up in balance (not even as pending).
After a hard crash on thursday I had to completly uninstall chia (including credentials).
After a (much faster, thx chia :)) resync my old coins showed up correclty with the old receiving address. But the new farmed one with the new receiving address does not.
Any suggestions?

**Comment by tiny0little** [comment]
try to research on [chiaexplorer.com](https://www.chiaexplorer.com/) or post your receiving address, I'll look

**Reply by atg906** [reply]
thx. seems like it was one of those self-transactions due to reset/etc...

**Comment by BrandtH22** [comment]
@atg906 , based on your previous message it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
